src/scss/core/components/_qty-input.scss

```diff
@@ -1,39 +1,71 @@
 .quantity-button {
+  display: flex;
+
   input {
-    max-width: 5rem;
+    z-index: 2;
+    width: 8rem;
+    height: 2.5rem;
+    padding: 0 0.375rem;
+    text-align: center;
+    border: 1px solid $input-border-color;
+    @include border-radius(4px);
+
+    &.ready {
+      width: 4rem;
+      @include border-radius(0);
+
+      &::-webkit-outer-spin-button,
+      &::-webkit-inner-spin-button {
+        margin: 0;
+        appearance: none;
+      }
+
+      &[type="number"] {
+        appearance: textfield;
+      }
+    }
   }
 
   button {
     display: flex;
     align-items: center;
     justify-content: center;
-    height: 50%;
+    min-width: 2rem;
```

Comment:
```suggestion
    min-width: 2rem;
```
Do we really need this? what about a padding instead? So we don't enforce any sizes, just asking, I need to deep in the subject before my review get more values

Reply:
I tried to prevent flashing in size when document is not loaded, size of input is different before and after script loaded.
So spin-button size must be fixed for calculation.

Script not loaded (low speed connection or refresh):

![Screenshot 2022-02-14 at 13-57-36 Cart](https://user-images.githubusercontent.com/85633460/153846901-098f6cba-4fab-412d-a75d-948a2d48f455.png)

Script loaded:

![Screenshot 2022-02-14 at 13-57-04 Cart](https://user-images.githubusercontent.com/85633460/153847050-18b04cb5-2a57-42bc-b8cd-32b33d10b174.png)

Reply:
Do we even need bootstrap-touchspin in theme-refacto? Why not use BS input-group component and add functions to - and + buttons?

Reply:
We don't need it, and we don't want it I think so

Reply:
I think this is the goal of this PR, inputQuantity doesn't use it anymore, it was the only place where it was used, we can completely remove it

Reply:
There are a lot of wrong things in the pattern of this PR, we don't need to build these buttons using JS, they can be directly built with input-group components as @SharakPL said, it will avoid this flashing content